wiki/Installation/de.md

```diff
@@ -2,15 +2,15 @@
 outdated: true
 ---
 
-# Installation
+# Installation (Deutsch)
```

Comment:
I guess the danger of changing this is that you don't know at a glance if this is actually a translation (loan word) or English. in this case it was already translated
```suggestion
# Installation
```
I think this is same for french, maybe more

Reply:
you're correct, It's the same word in french and swedish as well